x-pack/platform/plugins/shared/logs_shared/common/locators/logs_locator.ts

```diff
@@ -36,17 +35,8 @@ export class LogsLocatorDefinition implements LocatorDefinition<LogsLocatorParam
       this.deps.locators.get<DiscoverAppLocatorParams>('DISCOVER_APP_LOCATOR')!;
 
     return discoverAppLocator.getLocation({
-      dataViewSpec: params.dataViewSpec ?? (await this.getLogSourcesDataViewSpec()),
+      dataViewId: 'discover-observability-solution-all-logs',
```

Comment:
Can you use the [ALL_LOGS_DATA_VIEW_ID](https://github.com/elastic/kibana/blob/2673b9b93be5be8bf86d7f651ce864fa5f5fd39f/src/platform/packages/shared/kbn-discover-utils/src/data_types/logs/utils/get_all_logs_data_view_spec.ts#L12) constant here?

Reply:
Addressed in [6276a53](https://github.com/elastic/kibana/pull/211549/commits/6276a5366321b37991154475fbb6efdee62fc316).